analyzers/rspec/vbnet/S5753_vb.net.html

```diff
@@ -39,6 +39,8 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>See</h2>
 <ul>
+  <li> <a
```

Comment:
VB.NET version is missing the `web.config` part from C#

Reply:
Yes, these were added as comments 😄. I've now updated the spec.